redash/tasks/general.py

```diff
@@ -60,3 +62,33 @@ def send_mail(to, subject, html, text):
 
 def sync_user_details():
     users.sync_last_active_at()
+
+
+def purge_failed_jobs():
+    jobs = rq_redis_connection.scan_iter('rq:job:*')
+
+    is_idle = lambda key: rq_redis_connection.object('idletime', key) > settings.JOB_DEFAULT_FAILURE_TTL
```

Comment:
>  This subcommand is available when maxmemory-policy is set to an LRU policy or noeviction. 

(This command being `idletime`)

Is this the default config for Redis?

Reply:
The default is `noeviction`. On AWS, for example, it defaults to `volatile-lru`.